### DISCUSSION TITLE
Drag resize window, panes do not stay equal size

### DISCUSSION THREAD

**Discussion by esn89** [wezterm/wezterm#4399] [Q&A]
### What Operating System(s) are you running on?

macOS

### Which Wayland compositor or X11 Window manager(s) are you using?

N/A

### WezTerm version

wezterm 20230712-072601-f4abf8fd

### Ask your question!

I am able to split my wezterm into 2 panes.  I just use default settings so it splits it 50:50 in size.

However, when I drag and resize with my mouse on the top right, one of the panes become noticably smaller than the other.  Any ideas?

**Comment by raylu** [comment]
I think this is a dupe of #2865
